contributed_definitions/nyaml/NXgas_environment.yaml

```diff
@@ -0,0 +1,43 @@
+category: base
+doc: |
+  Gas phase environment around a sample.
+symbols:
+  n_components: |
+    number of gas phase components
+  n_pressure: |
+    number of values in gas phase pressure
+type: group
+NXgas_environment(NXexternal_stimulus):
+  gases(NX_CHAR):
+    doc: List of constituent gases.
+  density(NX_FLOAT):
+    unit: NX_MASS_DENSITY
+    doc: Overall density of the gas phase
+  concentration(NX_FLOAT):
+    unit: NX_UNITLESS
```

Comment:
no, a concentration is 1/AREA or 1/VOLUME so the unit is NX_ANY until we have no better way to write out unit categories in NeXus directly